### PR TITLE
feat(documentation): Add Cloudflare Logpull

### PR DESCRIPTION
**Pull request by PierrickV**

### CODE REVIEW

**Review by squioc — CHANGES_REQUESTED** [review]
Auto-generated file were embedded in the PR. I think they should be pushed in another PR.
Please update mkdocs.yml to point to your documentation, otherwise the user will be unable to access it

File: docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare_logpull.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Overview
+
+Cloudflare is a global network designed to make everything you connect to the Internet secure, private, fast, and reliable.
+
+In this documentation we will explain how to collect and send Cloudflare HTTP logs to SEKOIA.IO.
+
+{!operation_center/integration_catalog/generated/cloudflare_do_not_edit_manually.md!}
+
+## Configure
+
```
Comment (squioc) [inline docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare_logpull.md:14]:
```suggestion

### Create the intake

Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the format Cloudflare.

# Pull the events
```

File: docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare_logpull.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- `Cloudflare authentication key`
+- `Cloudflare Zone ID`
+- `Cloudflare account email`
+
+
+You now have all information to configure the Cloudflare module and its `Send events` action to SEKOIA.IO.
+
+You can now enable Cloudflare log retention (see [associated documentation](https://developers.cloudflare.com/logs/logpull/enabling-log-retention/)). 
+
+To start your playbook with a template, go to: "Create a new playbook" > "Use a template" > Search for Cloudflare.
```
Comment (squioc) [inline docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare_logpull.md:28]:
```suggestion
To pull events, go to [the playbook page](https://app.sekoia.io/operations/playbooks) and create your playbook with a template: "Create a new playbook" > "Use a template" > Search for Cloudflare.
```

File: mkdocs.yml [hunk 3]
```diff
@@ -53,6 +53,8 @@ nav:
         - Proxy: operation_center/integration_catalog/cloud_and_saas/cisco_umbrella/umbrella_proxy.md
         - IP: operation_center/integration_catalog/cloud_and_saas/cisco_umbrella/umbrella_ip.md
         - DNS: operation_center/integration_catalog/cloud_and_saas/cisco_umbrella/umbrella_dns.md
+      - Cloudflare:
```
Comment (squioc) [inline mkdocs.yml:56]: Could you also add the following entry:
```
      integrations/cloudflare.md: operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare_logpull.md
```
in the section `plugins > redirects > redirect_maps`.
Thanks

**Review by squioc — CHANGES_REQUESTED** [review]
Some suggestions and a missing redirect

**Review by squioc — DISMISSED** [review]

File: docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare_logpull.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+uuid: 0ba58f32-7dba-4084-ab17-90c0be6b1f10
+name: Cloudflare
+type: intake
+
+## Overview
+
+Cloudflare is a global network designed to make everything you connect to the Internet secure, private, fast, and reliable.
+
+In this documentation we will explain how to collect and send Cloudflare HTTP logs to SEKOIA.IO.
```
Comment (ka0ula) [inline docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare_logpull.md:9]:
```suggestion
In this documentation, you will learn how to collect and send Cloudflare HTTP logs to SEKOIA.IO.
```

**Review by squioc — APPROVED** [review]

**Pull request merged**